src/terms.js

```diff
@@ -232,30 +240,32 @@ export default [
     'holistic',
     'ideathon',
     'ideation',
-    'impactful',
+    'impact(ful|s)?',
     'impeccable',
     'in( |-)a( |-)nutshell',
     'incent',
-    'incentivi(s|z)e',
+    '(dis)?incentivi(s|z)e',
     'increase the odds',
     'incredibl(e|y)',
     '(inflat|redeem)able value',
     'influencers?',
     'influx',
+    'initiat(es?|ing|ives?)',
     'innovat(e|ed|ion|ive|ing|or)s?',
+    'in order',
```

Comment:
Might be too generic — e.g. in a sentence with `in order to`, replacing with "bullshit" will make the sentence nonsensical.

Reply:
Yeah, what the government means is to simply erase the words "in order". This kind of change may not be appropriate for bullshit.js